core/ui/src/components/applications-center/AddNoteModal.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <NsModal
+    :visible="visible"
+    :primaryButtonLabel="$t('common.save')"
```

Comment:
`primaryButtonLabel` and `secondaryButtonLabel` props do not exist in NsModal, let's remove them
```suggestion
```